### ISSUE TITLE
WINE with the fluent theme, error message but everything works

### ISSUE THREAD

**Issue by 4321ba** [OpenNBS/NoteBlockStudio#324]
**Describe the bug**
I'm running ONBS on linux with WINE (compatibility layer for windows executables on linux) and it works pretty well, but with the new fluent theme it displays an error message twice before launch. Despite that I don't experience any problems. I'm running it with a 32 bit wine prefix thing.

**To Reproduce**
I run ONBS with this command:
`WINEPREFIX=~/.wine32 WINEARCH="win32" wine Minecraft\ Note\ Block\ Studio.exe`
It happens with the latest stable version (3.9.3) as well as the latest beta (3.10.0 beta 1)
It only happens with the fluent theme, if I start it with any other theme, the error doesnt appear. Also if I start it with another theme and switch to fluent, it appears (though only once this time?). (tested this with the beta)
when I click OK, everything works

**Expected behavior**
no error appears

**Screenshots**
![Screenshot_20220409_191249](https://user-images.githubusercontent.com/39439556/162584566-670a1a47-b78c-4dde-a48e-bad458ca715c.png)
(the window title is "Error")
**Additional context**
Wine version (ancient probably): wine-5.0 (Ubuntu 5.0-3ubuntu1)
edit: tested with a more recent version, in [Bottles](https://flathub.org/apps/details/com.usebottles.bottles), with caffe-7.5 (whatever that means)
OS: Ubuntu 20.04.4 LTS x86_64
Kernel: 5.4.0-107-generic

this is like a really low priority thing, I just find the new theme very nice, and use it, but now it's a bit annoying pressing space bar twice when I launch ONBS :D.

**Comment by chenxi050402** [comment]
the popup is removed in current versions, probably resolved
